Prog/vacant-lots-montgomery_YS.Rmd

```diff
@@ -0,0 +1,1257 @@
+
+
+map_df(c(.7, .85), ~ get_unit_summary_r(dataset = zoningdata_rockvile1_final,
+                                      construction_efficiency = .,
+                                      average_unit_size = 940,
+                                      dwelling_type_breakdown = FALSE)) %>% 
+  knitr::kable()
+
+```
+
+Look at the distribution of units for lots with more than 1 unit, to check for extreme outliers.
+
+```{r scatter Rockville, echo = FALSE}
+
+zoningdata_rockvile1_final %>% 
+  get_unit_variables_r(construction_efficiency = .7,
+                     average_unit_size = 940) %>% 
+  filter(units_total > 1) %>% 
+  ggplot() +
+  geom_point(mapping = aes(lotsize_sf, units_total),
+             alpha = .5) +
+  scale_x_continuous(labels = scales::comma,
+                     expand = expand_scale(mult = c(0, 0.002)), 
+                     limits = c(0, 300000),
+                     breaks = 0:6 * 50000) +
+  scale_y_continuous(expand = expand_scale(mult = c(0, 0.002)), 
+                     limits = c(0, 300),
+                     breaks = 0:7 * 50) +
+  scatter_grid() +
+  labs(x = "Lot size (square feet)", y = "Total units") +
+  theme(plot.margin = margin(t = 20, r = 20))
+
+
+```
+
+
+Look more closely at vacant lots < 50,000 SF
+
+```{r scatter-small Rockville, echo = FALSE}
+
+zoningdata_rockvile1_final %>% 
+  get_unit_variables_r(construction_efficiency = .7,
+                     average_unit_size = 940) %>% 
+  group_by(housing_type) %>% 
+  count() %>% 
+  knitr::kable()
+
+zoningdata_rockvile1_final %>% 
+  get_unit_variables_r(construction_efficiency = .7,
+                     average_unit_size = 940) %>% 
+  filter(lotsize_sf < 50000,
+         units_total > 1) %>% 
+  ggplot() +
+  geom_point(mapping = aes(lotsize_sf, units_total,
+                           color = housing_type),
+             alpha = .8) +
+  scale_x_continuous(labels = scales::comma,
+                     expand = expand_scale(mult = c(0, 0.0002)), 
+                     limits = c(0, 50000),
+                     breaks = 0:5 * 10000) +
+  scale_y_continuous(expand = expand_scale(mult = c(0, 0.0002)), 
+                     limits = c(0, 60),
+                     breaks = 0:4 * 15) +
+  scatter_grid() +
+  labs(x = "Lot size (square feet)", y = "Total units") +
+  theme(plot.margin = margin(t = 20,
+                             r = 20))
+
+
+
+```
+
+## Write out results for rockville for merging with Montgomery
+
+Write out the results to a csv. Include jurisdiction name in the title.
+
+```{r write-results Rockville}
+
+#document blacknight number of units
+zoningdata_rockvile1_final2 <- get_unit_variables_r(dataset =zoningdata_rockvile1_final,
+                                                    construction_efficiency = .7,
+                                                     average_unit_size = 940) %>% 
+                     mutate(unit_bk= ifelse(category=="sf", 1, numberofunits))
+
+
+Rockvillevacant <- zoningdata_rockvile1_final2 %>% 
```

Comment:
does this incorporate the assessment values to select out vacant with structure?

Reply:
yes